tests/unit/stylo/build.rs

```diff
@@ -2,24 +2,67 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+extern crate regex;
+
+use regex::Regex;
 use std::env;
-use std::fs;
-use std::io::Write;
-use std::path;
-use std::process::Command;
+use std::fs::File;
+use std::io::{BufRead, BufReader, Write};
+use std::path::Path;
 
 fn main() {
+    let root_path = Path::new("../../../");
+    let bindings_file = root_path.join("components/style/gecko/generated/bindings.rs");
+    let glue_file = root_path.join("ports/geckolib/glue.rs");
+
     println!("cargo:rerun-if-changed=build.rs");
-    println!("cargo:rerun-if-changed=check_bindings.py");
-    println!("cargo:rerun-if-changed=../../../ports/geckolib/glue.rs");
-    println!("cargo:rerun-if-changed=../../../components/style/gecko_bindings/bindings.rs");
-    assert!(Command::new("python").arg("./check_bindings.py")
-                                  .spawn().unwrap().wait().unwrap().success());
+    println!("cargo:rerun-if-changed={:?}", glue_file);
```

Comment:
Because I was being lazy.  You're right, though, `Display` is more likely to give good results here than `Debug`, will fix.